### PR TITLE
adds new access type relativeOciReference

### PR DESCRIPTION
**Pull request by jschicktanz**
**What this PR does / why we need it**:
adds new access type relativeOciReference

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement user
- adds new access type relativeOciReference
```

### CODE REVIEW

File: bindings-go/apis/v2/accesstypes.go [hunk 1]
```diff
@@ -69,6 +69,46 @@ func (O *OCIRegistryAccess) SetData(bytes []byte) error {
...
 
+// RelativeOciReferenceType is the access type of a relative oci reference.
+const RelativeOciReferenceType = "relativeOciReference"
+
+// RelativeOciAccess describes the access for a relative oci reference.
+type RelativeOciAccess struct {
+	ObjectType `json:",inline"`
+
+	// Reference is the relative reference to the oci image repository and tag.
+	// The format is expected to be "repository:tag".
```
Comment (schrodit) [inline bindings-go/apis/v2/accesstypes.go:80]: `repository@digest` should also be supported?

**Review by schrodit — CHANGES_REQUESTED** [review]
just minor nit otherwise lgtm

File: bindings-go/apis/v2/accesstypes.go (hunk 1 above)
Comment (jschicktanz) [reply bindings-go/apis/v2/accesstypes.go:80]: afaik the tooling currently has no support for the repository@digest format, same goes for the regular ociRegistry access type

**Review by schrodit — APPROVED** [review]
/lgtm

**Pull request merged**